src/thunderbots/software/radio_communication/visitor/mrf_primitive_visitor.cpp

```diff
@@ -98,6 +98,8 @@ void MRFPrimitiveVisitor::visit(const MovePrimitive &move_primitive)
         move_primitive.getFinalAngle().toRadians() * CENTIRADIANS_PER_RADIAN,
         move_primitive.getFinalSpeed() * MILLIMETERS_PER_METER};
     radio_prim->extra_bits = 0;
+    radio_prim->extra_bits += move_primitive.getAutoKickEnabled() * 0x01;
```

Comment:
I'd prefer to use `|=` here to show we're setting bits. Otherwise I can see the addition becoming confusing and potentially causing weird behavior if done wrong since it would add to the number and cause all bits to change.

Reply:
Changed!